### PR TITLE
[FIX] portal_rating: keep comment section on canceling edition

### PR DESCRIPTION
**Pull request by adsh-odoo**
Steps to Reproduce
===================
1 Open any course
2 Add a review / or edit the review
3 Go to the Reviews tab. (you will find a comment button under each review)
4 Post a comment on any one review.
5 Now from the 3 dots, click on Edit
6 Now, your mind has changed and you don't want to edit anything.
  Click on the "Cancel" button.
The whole comment section disappears, including the existing comment if any, and
the 'comment' button on the parent comment.

Technical
=========
This issue was introduced after the removal of qweb https://github.com/odoo/odoo/commit/6303a3eacdca012649a2ffda627b65c17a7217f1

After this commit
==================
The comments section appears properly after canceling the edition.

Task-3642425

### CODE REVIEW

File: addons/portal_rating/static/src/js/portal_chatter.js
Comment (Odoonan) [inline addons/portal_rating/static/src/js/portal_chatter.js:1]: CM guidelines, maybe rephrase to have a verb first '[FIX] portal_rating: keep comment section on canceling edition'

And then at the end of steps add something like 'The whole comment section disappear, including the existing comment if any, and the 'comment' button on the parent comment.' to complete :)

File: addons/portal_rating/static/src/js/portal_chatter.js [hunk 1]
```diff
@@ -346,13 +346,14 @@ PortalChatter.include({
...
         if (comment) {
             var data = {
                 rating: this.messages[messageIndex].rating,
                 is_publisher: this.options.is_user_publisher,
             };
-            this._getCommentContainer($source).append(renderToElement("portal_rating.chatter_rating_publisher_comment", data));
+            this._getCommentContainer($source).empty().append(renderToElement("portal_rating.chatter_rating_publisher_comment", data));
+        } else {
+            this._getCommentContainer($source).empty();
         }
```
Comment (Odoonan) [inline addons/portal_rating/static/src/js/portal_chatter.js:357]: Ah yes I see, as _getCommentContainer is relative, when emptying it beforehand, it does not retrieve the element. LGTM :)

**Review by Odoonan — COMMENTED** [review]
Hello @adsh-odoo , just a small comment on the commit message. LGTM otherwise. Thanks and cheers! You can green once the CM is updated :)

File: addons/portal_rating/static/src/js/portal_chatter.js [hunk 2]
```diff
@@ -346,13 +346,14 @@ PortalChatter.include({
         var messageIndex = $source.data("mes_index");
```
Comment (awa-odoo) [inline addons/portal_rating/static/src/js/portal_chatter.js:346]: Commit message:

Let's add a reference to the commit that introduced the issue, I believe it's https://github.com/odoo/odoo/commit/6303a3eacdca012649a2ffda627b65c17a7217f1

File: addons/portal_rating/static/src/js/portal_chatter.js [hunk 3]
```diff
@@ -346,13 +346,14 @@ PortalChatter.include({
...
 
         var comment = this.messages[messageIndex].rating.publisher_comment;
-        this._getCommentContainer($source).empty();
```
Comment (awa-odoo) [inline addons/portal_rating/static/src/js/portal_chatter.js:349]: Wouldn't it be a bit cleaner to store the comment container into a variable and then use it to do the operations?

So:

```
const $commentContainer = this._getCommentContainer($source);
$commentContainer.empty();
if (comment) {
    ...
    $commentContainer.append(...);
}
```

**Review by awa-odoo — COMMENTED** [review]
Hello @adsh-odoo 

Some light comments but globally LGTM.

Thanks & cheers!

**Comment by adsh-odoo** [comment]
Hello @awa-odoo 

Suggested changes have been made.
Can you please have a look?

Thanks!!

**Comment by awa-odoo** [comment]
@robodoo r+
